{"name": "test_delete_added_request", "status": "broken", "statusDetails": {"message": "AttributeError: 'list' object has no attribute 'find_elements_by_tag_name'", "trace": "self = <test_case_details.TestCaseDetails object at 0x0D95C710>\n\n    def test_delete_added_request(self):\n        driver = self.driver\n        self.login()\n        case_details = CaseDetails(driver)\n        home = HomePage(driver)\n        home.click_law_suit_page()\n        case_details.request_tab_entry()\n>       case_details.delete_added_request()\n\ntest_case_details.py:87: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <Pages.CaseDetailsPage.CaseDetails object at 0x0D95CA90>\n\n    def delete_added_request(self):\n        table = self.driver.find_elements_by_xpath(self.directive_table_xpath) # tbody\n>       tds = table.find_elements_by_tag_name(\"td\")\nE       AttributeError: 'list' object has no attribute 'find_elements_by_tag_name'\n\n..\\Pages\\CaseDetailsPage.py:176: AttributeError"}, "start": 1568629681880, "stop": 1568629701538, "uuid": "db908a35-8c9c-4bbe-8a96-bfe67de21e11", "historyId": "812199f097f3799c533dced022272d50", "testCaseId": "546de6c208ed1c45d32a4ab8f8a947c2", "fullName": "test_case_details.TestCaseDetails#test_delete_added_request", "labels": [{"name": "parentSuite"}, {"name": "suite", "value": "test_case_details"}, {"name": "subSuite", "value": "TestCaseDetails"}, {"name": "host", "value": "EVT01066NB"}, {"name": "thread", "value": "15208-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case_details"}]}